src/runtime/config/configuration-qemu-snp.toml.in

```diff
@@ -76,13 +82,13 @@ kernel_params = "@SNPKERNELPARAMS@"
 
 # Path to the firmware.
 # If you want that qemu uses the default firmware leave this option empty
-firmware = "@SNPFIRMWAREPATH@"
+firmware = "@FIRMWARESNPPATH@"
 
 # Path to the firmware volume.
 # firmware TDVF or OVMF can be split into FIRMWARE_VARS.fd (UEFI variables
 # as configuration) and FIRMWARE_CODE.fd (UEFI program image). UEFI variables
 # can be customized per each user while UEFI code is kept same.
-firmware_volume = "@FIRMWAREVOLUMEPATH@"
+firmware_volume = "@FIRMWARETDVFVOLUMEPATH@"
```

Comment:
We should not be using the TDVF volume here.

With SEV/SNP we're actually not supposed to be using the firmware volume at all, but it looks like we have been. I guess you can keep it as `FIRMWAREVOLUMEPATH` for now and I will see if we can comment it out entirely later.

Reply:
Noted, is this a bug in main as well?

Reply:
Ok turns out `FIRMWAREVOLUMEPATH` and `FIRMWARETDVFVOLUMEPATH` are both empty. It looks like we use the wrong one in main tho. I will tweak that at some point.